requirements.txt

```diff
@@ -3,7 +3,7 @@ pycryptodomex>=3.7.0
 requests>=2.9.1
 vdf>=3.3
 gevent>=1.3.0
-protobuf>=3.0.0
+protobuf>=3.0.0,<3.18.0
```

Comment:
This should probably be:

```
protobuf>~3.0
protobuf<3.18; python_version < '3'
```
Not sure if this work 🤔

Reply:
Done. That gave me a "Double requirement given" error with Python 2.7, but I was able to work around it by adding a `python_version` marker to both lines.

Reply:
Nice that works